src/util/AbstractAPI.js

```diff
@@ -107,12 +107,18 @@ Ext.define('Jarvus.util.AbstractAPI', {
                 if (response.aborted === true) {
                     Ext.callback(options.abort, options.scope, [response]);
                 } else if (response.status == 401 || response.statusText.indexOf('Unauthorized') !== -1) {
-
+                    
                     /*
                     We seem to always get the same session id, so we can't automatically try again once the user logs in
                     var oldSessionID = Ext.util.Cookies.get('s');
                      */
 
+                    // Send user to callback if login attempt has failed
+                    if (options.url == '/login') {
+                        Ext.callback(options.success, options.scope, [response]);
```

Comment:
The overall goal here is to display a login form if the user isn't logged in or load the interface if they are. I'm using API.loadSessionData() to check their status however, if they aren't logged in, a message pops up saying 'Login Required' and no other events are fired for me to hook into. 

See OntrayPOS: https://github.com/JarvusInnovations/ontray/blob/mobile/sencha-workspace/OntrayPOS/app/controller/Viewport.js#L52

I just don't think we should force this Login Required message on any unauthenticated call.

https://github.com/JarvusInnovations/jarvus-apikit/blob/master/src/util/AbstractAPI.js#L127-L151